pyomo/repn/plugins/baron_writer.py

```diff
@@ -70,6 +111,7 @@ def visit(self, node, values):
                 if val and val[0] in '-+':
                     parens = True
                 elif arg.__class__ in native_numeric_types:
+                    #assert type(val) is str
```

Comment:
Is this a debugging statement that can be removed?

Reply:
I am not even sure that it is debugging (I am pretty sure that assertion would always fail).  It can definitely be deleted.